src/storage/src/hummock/compactor.rs

```diff
@@ -451,6 +457,8 @@ impl Compactor {
         let get_id_time = Arc::new(AtomicU64::new(0));
 
         // NOTICE: should be user_key overlap, NOT full_key overlap!
+        // TODO: refine resource size
+        let resource_size = (self.context.options.sstable_size_mb * 1024 * 1024) as u64;
```

Comment:
If you only allow resource limit with `sstable_size_mb`, it would cause that there will be only one thread could execute compaction.

Reply:
Emm, I made a mistake. Ignore it.

Reply:
`resource_size` is the size requested from global quota. The global quota is defined here
```
            // TODO: load from config
            sst_builder_limiter: Arc::new(ResourceLimiter::new(1024 * 1024 * 1024)),
```

Reply:
Anyway this fixed `resource_size` is not optimal. For example we are building 4 tiny SSTs and have requested 4 x 256MB resource, which prevents another SST to be built at the same time.